### PR TITLE
Return cache headers for location pages

### PR DESCRIPTION
**Pull request by andrewgarner**

### CODE REVIEW

File: config/initializers/locations.rb [hunk 1]
```diff
@@ -1 +1,5 @@
-Locations.geo_json_path_or_url = ENV['LOCATIONS_URL'] || 'https://locator.pensionwise.gov.uk/locations.json'
+Locations.geo_json_path_or_url = if Rails.env.test?
+                                   Rails.root.join('features', 'fixtures', 'locations.json')
```
Comment (stevenwilkin) [inline config/initializers/locations.rb]: This is set in [env.rb](https://github.com/guidance-guarantee-programme/pension_guidance/blob/6a3173d00c69c97feab8dfe95e210ab1ac12d128/features/support/env.rb#L6)

Comment (andrewgarner) [inline config/initializers/locations.rb]: That's only applicable for the acceptance tests.

Comment (stevenwilkin) [inline config/initializers/locations.rb]: We should set it only once then? Removing it from `env.rb` would probably be the thing to do

Comment (andrewgarner) [inline config/initializers/locations.rb]: :+1:

Comment (andrewgarner) [inline config/initializers/locations.rb]: @stevenwilkin - done and rebased.

**Pull request merged**